docs/release-notes.md

```diff
@@ -1,101 +1,81 @@
+    For those in the United States, I hope you all had a wonderful Thanksgiving holiday and for our customers everywhere, our team is very thankful for your feedback, contributions, and the opportunity to continue working with all of you.  You all are the reason we are able to continue working on Open AMT Cloud Toolkit.  We have implemented some much asked for improvements and features in our November release in addition to a whole bevy of resolved issues and external contributions!  Check out the [release video](#release-highlights) where Mike talks about this release's highlights and you can find the details of this release under [what's new](#whats-new) section.
 
     *Best wishes,*  
     *The Open AMT Cloud Toolkit Team*
 
 
 ## What's New?
 
+:material-star:** Database update required **
+An upgrade to the `mpsdb` database is required. Please run the following SQL script to add two new columns before upgrading the services:
+``` sql
+ALTER TABLE devices 
+ADD COLUMN IF NOT EXISTS friendlyname varchar(256);
+ALTER TABLE devices 
+ADD COLUMN IF NOT EXISTS dnssuffix varchar(256);
+```
+
 :material-new-box:** Feature: AMT Maintenance commands **
 
-Three new maintenance commands have been added to RPC and RPS to support syncing the AMT Clock, syncing the IP settings, and changing the AMT password.
+We have added a maintenance task that will enable syncing the OS Hostname and DNS Suffix with AMT Hostname and DNS Suffix.  When this command is run, if the OS and AMT host names are different, the OS hostname will be applied to AMT.  This enables AMT to respond to network requests that use the fully qualified domain name (FQDN) of the device.
 
-:material-star:** Customer Request: Unprovision AMT devices via MPS **
+:material-star:** Customer Request: Service startup improvement **
 
-We've had several customer requests to add this feature to MPS and this release we are delivering.  We've added an API endpoint ( DELETE /api/v1/amt/deactivate/{guid} ) to MPS that will send the unprovision command to a connected AMT device and will then remove that AMT device from the MPS database.  In order to reconnect that AMT device to MPS again, RPC and RPS will need to be used.
+Both RPS and MPS will now check for dependent services to be available before starting up.  If a dependent service is not available, they will wait and retry after a few seconds.  If after 10 attempts the service will shutdown.  In a production deployment, this shutdown will typically trigger a restart of the service starting the cycle again until dependent services are available. Retry attempts will be listed in the service logs.
```

Comment:
Both RPS and MPS now check for dependent services available before starting up.  If a dependent service is not available, the RPS and MPS wait and retry after a few seconds.  After 10 retry attempts the service shuts down.  In a production deployment, this shutdown typically triggers a restart of the service, starting the cycle again until dependent services are available. Retry attempts are listed in the service logs.